packages/bpk-component-calendar/examples-components.js

```diff
@@ -21,7 +21,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import BpkButton from 'bpk-component-button';
-import addMonths from 'date-fns/add_months';
+import addMonths from 'date-fns/addMonths';
```

Comment:
2.x changed all imports to use camelCase instead of snake_case. There are a few places you see this throughout this PR.